### DISCUSSION TITLE
Wrong players position for POV Demo ?

### DISCUSSION THREAD

**Discussion by ghost** [saul/demofile#218] [Questions] [answered]
I recorded a demo from my POV. When I try to get other players position when an event is fired, I got mine, is that a correct behavior ?
```

 var player = demo.entities.getByUserId(e.userid);
            console.log(player.name);

            if (isWeaponMonitored(e.weapon) !== -1) {
                var position = player.position;
                console.log(position);
```

Displays : 

```
Agouha 
{ x: 1257.800048828125, y: -229.25465393066406, z: -163.96875 }

소리
{ x: 1159.30859375,  y: 45.68016052246094,  z: -167.10072326660156 }
```

While one player is at T spawn, and the other at CT spawn, it just displays my position at T spawn for both players.

**Comment by ghost** [comment]
Seems like it's a normal behavior. Not sure. I'm going to test with a gotv demo

**Comment by ghost** [comment]
Behavior seems normal, no problems with a GOTV demo !

**Comment by saul** [comment, marked as answer]
This should have been fixed with the recent improvement to POV demos.
